micro-jmx-metrics/build.gradle

```diff
@@ -3,7 +3,12 @@ description = 'micro-jmx-metrics'
 dependencies {
 
 
-    compile ('com.ryantenney.metrics:metrics-spring:'+springMetricsVersion){ exclude(module: 'org.springframework') }
+    compile ('com.ryantenney.metrics:metrics-spring:'+springMetricsVersion){
+        exclude(module: 'org.springframework')
+        exclude(group: 'io.dropwizard.metrics', module: 'metrics-core')
```

Comment:
Why did you have to exclude this?

Reply:
metrics-spring does not use the newer version of metrics-core (from dropwizard).
The intention here is to include metrics-spring exception for io.dropwizard.metrics and we explicitly include it from micro-metrics::build.gradle